text/0000-pack-subcommands.md

```diff
@@ -0,0 +1,275 @@
+# Meta
+[meta]: #meta
+- Name: `pack` subcommands
+- Start Date: 2020-07-08
+- Author(s): @jromero
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: "N/A"
+
+# Summary
+[summary]: #summary
+
+Update the `pack` CLI to use more common subcommand patterns.
+
+# Motivation
+[motivation]: #motivation
+
+As the number of subcommands for `pack` grow, it's becoming more difficult to identify interworking subcommands at first glance. See the current list of subcommands below.
+
+```
+pack
+
+  build                 Generate app image from source code
+  rebase                Rebase app image with latest run image
+  inspect-image         Show information about a built image
+  set-run-image-mirrors Set mirrors to other repositories for a given run image
+  set-default-builder   Set default builder used by other commands
+  inspect-builder       Show information about a builder
+  suggest-builders      Display list of recommended builders
+  trust-builder         Trust builder
+  untrust-builder       Stop trusting builder
+  list-trusted-builders List Trusted Builders
+  create-builder        Create builder image
+  package-buildpack     Package buildpack in OCI format.
+  suggest-stacks        Display list of recommended stacks
+  version               Show current 'pack' version
+  report                Display useful information for reporting an issue
+  completion            Outputs completion script location
+  help                  Help about any command
+```
+
+# What it is
+[what-it-is]: #what-it-is
+
+
+### Proposed structure
+
+The pack commands will be reorganized in a way where they are grouped by resource type where possible. Additionally, a new
+command `config` will group subcommands that solely operate on mutating the config file.
+
+
+An overview of the (sub)commands:
+
+```
+pack
+
+  build
+  builder
+    create
+    suggest
+  buildpack
+    package
+    register
+  completion
+  config
+    default-builder
+    run-image-mirrors
+      add
+      remove
+      list
+    registries
+      add
+      remove
+      list
+    trusted-builders
+      add
+      remove
+      list
+  help
+  inspect
```

Comment:
We currently have both `pack inspect-image` and `pack inspect-builder`. What would you envision with this `inspect` command?